contracts/sdjwt-verifier/src/types.rs

```diff
@@ -79,60 +86,97 @@ pub enum MathsOperator {
     EqualTo,
 }
 
+/// Validate the verified claims against the presentation request
 pub fn validate(
     presentation_request: PresentationReq,
     verified_claims: serde_json::Value,
-) -> Result<bool, SdjwtVerifierError> {
+    block_info: &BlockInfo,
+) -> Result<(), SdjwtVerifierResultError> {
     if let serde_json::Value::Object(claims) = verified_claims {
         for (key, criterion) in presentation_request {
-            if let Some(value) = claims.get(&key) {
+            // For every requested key, the value must be present in the verified claims
+            if key == CW_EXPIRATION && criterion == Criterion::Expires(true) {
+                if let Some(value) = claims.get(&key) {
```

Comment:
Please consider refactor it as follows:
```rust
match claims.get(&key) {
    Some(serde_json::Value::String(exp)) => {},
    Some(_) => {},
   None => {}
}
```

Reply:
fixed in https://github.com/nymlab/AVIDA/tree/2d03896591111ec65986b15568dd90fb23bf0b3e